### ISSUE TITLE
Проблема: `htmlacademy/aria-label-misuse` реагирует на `aria-label` для `<label>`

### ISSUE THREAD

**Issue by nikolai-shabalin** [htmlacademy/linthtml-rules-htmlacademy#72]
### Ошибка из терминала

```bash
160:19  error  "aria-label" cannot be used on this element  htmlacademy/aria-label-misuse
```

### Часть кода на котором произошла ошибка

```html
<li class="filter__item">
  <input class="visually-hidden filter__radio" id="bluetooth-exclude" name="bluetooth" type="radio" value="exclude">
  <label for="bluetooth-exclude" aria-label="функция не нужна">Нет</label>
</li>
```

**Comment by balesniy** [comment]
неожиданно) надо разрешить тогда в этом правиле и может сделать отдельное на примере https://www.aditus.io/aria/aria-label/#common-mistakes
